src/components/LeftNavigation/LeftNavigation.Props.ts

```diff
@@ -29,6 +28,7 @@ export interface ILeftNavigationOption {
     disabled?: boolean;
     position?: LeftNavigationOptionPositionEnum;
     notificationNumber?: number;
+    notificationBubbleStyleObject?: NotificationBubbleStyleObject;
```

Comment:
why not both? set one default style for the menu and be able to override it per menu item?

Reply:
Added both options.